packages/grafana-ui/src/components/Logs/LogDetailsRow.tsx

```diff
@@ -44,6 +47,9 @@ const getStyles = stylesFactory((theme: GrafanaTheme) => {
       label: wordBreakAll;
       word-break: break-all;
     `,
+    showingField: css`
+      color: ${theme.palette.blue95};
```

Comment:
@jessover9000 what's a good color here to highlight an icon as active (I've said blue) and how should the color be referenced?

Reply:
Example: <img width="225" alt="Screenshot 2020-08-18 at 15 14 57" src="https://user-images.githubusercontent.com/859729/90517391-9e2e4380-e165-11ea-9756-4dc39fef1af9.png">